features/frameworks/utest/utest/utest_shim.h

```diff
@@ -27,51 +27,13 @@
 #include <stdio.h>
 #include "utest/utest_scheduler.h"
 
-#ifdef YOTTA_CFG
-#   include "compiler-polyfill/attributes.h"
-#else
-#   ifndef __deprecated_message
```

Comment:
this one is still used - `__deprecated_message` - see Travis errors

Reply:
Oops, I had fixed this locally, just hadn't pushed it up, thanks!